about/press_kit/index.markdown

```diff
@@ -21,54 +23,91 @@ Specific version references of MoveIt framework:
 - All specific mentions of the MoveIt are written as "MoveIt 2", "MoveIt 3", etc.
 - All mentions of the MoveIt 2 port will be written as “MoveIt 2” or as “the MoveIt port to ROS 2.0"
 
-## Logo Guidelines
 
-We ask that you not:
+## Trademark Use ##
```

Comment:
```suggestion
# MoveIt Trademark Policy

The "MoveIt" name and other MoveIt trademarks are property of PickNik Inc. ("PickNik" or "PickNik Robotics").

Below are the relevant agreements, rules, and guidelines on displaying the MoveIt trademarks, which may be updated from time to time. Use of any MoveIt trademark must be in accordance with this Policy or a separate written agreement between you and PickNik.

## Trademark Use ##
```

It was pointed out we forgot to copy this part from [OSRF's page](https://www.ros.org/press-kit/), but it seems like important context actually.